client/lib/typeform.ts

```diff
@@ -0,0 +1,40 @@
+// See https://github.com/jeremenichelli/typeform-payload-helpers
+
+export const refsMap = {
```

Comment:
Maybe add a comment about how you got these values? It's gonna be mental to have to re-derive how we did this all.